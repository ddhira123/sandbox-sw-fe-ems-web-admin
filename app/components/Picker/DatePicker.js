import { DatePicker as DtPick } from 'antd';
import styled from 'styled-components';

const DatePicker = styled(DtPick)`
  border-radius: 10px !important;
  height: 40px !important;
`;

export default DatePicker;
