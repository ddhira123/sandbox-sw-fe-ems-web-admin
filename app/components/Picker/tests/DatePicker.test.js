import React from 'react';
import { render } from '@testing-library/react';
import DatePicker from '../DatePicker';

describe('DatePicker', () => {
  it('should render correctly', () => {
    const app = render(<DatePicker />);
    expect(app).toMatchSnapshot();
  });
});
