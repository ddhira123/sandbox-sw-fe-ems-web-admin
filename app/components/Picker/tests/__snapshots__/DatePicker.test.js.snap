// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DatePicker should render correctly 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="ant-picker DatePicker-g17zbi-0 eYrOtU"
      >
        <div
          class="ant-picker-input"
        >
          <input
            autocomplete="off"
            placeholder="Select date"
            readonly=""
            size="12"
            title=""
            value=""
          />
          <span
            class="ant-picker-suffix"
          >
            <span
              aria-label="calendar"
              class="anticon anticon-calendar"
              role="img"
            >
              <svg
                aria-hidden="true"
                class=""
                data-icon="calendar"
                fill="currentColor"
                focusable="false"
                height="1em"
                viewBox="64 64 896 896"
                width="1em"
              >
                <path
                  d="M880 184H712v-64c0-4.4-3.6-8-8-8h-56c-4.4 0-8 3.6-8 8v64H384v-64c0-4.4-3.6-8-8-8h-56c-4.4 0-8 3.6-8 8v64H144c-17.7 0-32 14.3-32 32v664c0 17.7 14.3 32 32 32h736c17.7 0 32-14.3 32-32V216c0-17.7-14.3-32-32-32zm-40 656H184V460h656v380zM184 392V256h128v48c0 4.4 3.6 8 8 8h56c4.4 0 8-3.6 8-8v-48h256v48c0 4.4 3.6 8 8 8h56c4.4 0 8-3.6 8-8v-48h128v136H184z"
                />
              </svg>
            </span>
          </span>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="ant-picker DatePicker-g17zbi-0 eYrOtU"
    >
      <div
        class="ant-picker-input"
      >
        <input
          autocomplete="off"
          placeholder="Select date"
          readonly=""
          size="12"
          title=""
          value=""
        />
        <span
          class="ant-picker-suffix"
        >
          <span
            aria-label="calendar"
            class="anticon anticon-calendar"
            role="img"
          >
            <svg
              aria-hidden="true"
              class=""
              data-icon="calendar"
              fill="currentColor"
              focusable="false"
              height="1em"
              viewBox="64 64 896 896"
              width="1em"
            >
              <path
                d="M880 184H712v-64c0-4.4-3.6-8-8-8h-56c-4.4 0-8 3.6-8 8v64H384v-64c0-4.4-3.6-8-8-8h-56c-4.4 0-8 3.6-8 8v64H144c-17.7 0-32 14.3-32 32v664c0 17.7 14.3 32 32 32h736c17.7 0 32-14.3 32-32V216c0-17.7-14.3-32-32-32zm-40 656H184V460h656v380zM184 392V256h128v48c0 4.4 3.6 8 8 8h56c4.4 0 8-3.6 8-8v-48h256v48c0 4.4 3.6 8 8 8h56c4.4 0 8-3.6 8-8v-48h128v136H184z"
              />
            </svg>
          </span>
        </span>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
